service/src/main/kotlin/no/nav/su/se/bakover/service/revurdering/RevurderingServiceImpl.kt

```diff
@@ -251,19 +248,20 @@ internal class RevurderingServiceImpl(
         val revurdering = revurderingRepo.hent(revurderingId)
             ?: return KunneIkkeHenteGjeldendeGrunnlagsdataOgVilkårsvurderinger.FantIkkeBehandling.left()
 
-        val sak = sakService.hentSak(revurdering.sakId).getOrHandle {
-            return KunneIkkeHenteGjeldendeGrunnlagsdataOgVilkårsvurderinger.FantIkkeSak.left()
-        }
-
-        return KopierGjeldendeGrunnlagsdataOgVilkårsvurderinger(
-            periode = revurdering.periode,
-            vedtakListe = sak.vedtakListe,
-        ).let {
-            HentGjeldendeGrunnlagsdataOgVilkårsvurderingerResponse(
-                it.grunnlagsdata,
-                it.vilkårsvurderinger,
-            )
-        }.right()
+        return sakService.kopierGjeldendeVedtaksdata(revurdering.sakId, revurdering.periode.fraOgMed)
```

Comment:
Det ser ut som vi kun persisterer ting som ligger i OpprettetRevurdering som blir instansiert over. Er det da noen grunn til at vi må hente fra databasen på nytt? Er det noe data som blir generert/populert annerledes når vi henter på nytt? I så fall synes jeg vi kan ha en liten linje eller to med forklaring her.

Reply:
Nevermind. Expandet koden og ser at dette er en egen funksjon :] Passet veldig fint sammen dog!